### PR TITLE
use same nuget key as main serilog repo

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Comment by SimonCropp** [comment]
@nblumhardt is there a reason why the nuget keys are different on the diff repos?

File: appveyor.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ artifacts:
 deploy:
 - provider: NuGet
   api_key:
-    secure: kYR3BYzJm3wSFbFjPhgTzuDHHcE8ApoNUmHvJvunWZ39nyrqVk6J6srjzYVQ7/gX
+    secure: 45/vGyNCdoOvWSorcVX6qYM3oC/mCBj0CDRXNZP4twlIrBiZ9sKtKMdHwufm4ogS
```
Comment (sungam3r) [inline appveyor.yml:14]: How does it work in terms of safety? Anyone can send their package now?

Comment (SimonCropp) [reply appveyor.yml:14]: that key should be one replaced by the encrypted one from appveyor during a build. 
@nblumhardt can u confirm?

Comment (nblumhardt) [reply appveyor.yml:14]: Appveyor has the private key to decrypt this at build time (it's base-64 AES-256 cyphertext).

**Comment by nblumhardt** [comment]
I think the core Serilog package needs/uses a different key from the sinks/extensions because (historically) the contributor/maintainer group for the core was much more limited than for the additions and this helped firewall the supply chain just a little bit more :-)

Right now, this will fail if merged, but given the much smaller Serilog org we now have, it would make sense to either use a single key across them, or even keep a key per repository 🤔

**Comment by nblumhardt** [comment]
(Marking as draft helps avoid an accidental later merge :-))

**Comment by SimonCropp** [comment]
@nblumhardt i only did this since current key has expired https://ci.appveyor.com/project/serilog/serilog-framework-logging/builds/46479990

can u gen a new key and update the one stored in appveyor

**Comment by nblumhardt** [comment]
@SimonCropp thanks 👍 - `EN9f+XXE3fW+ebL4wxrIbafdtbNvRfddBN8UUixvctYh4qMBHzr1JdnM83QsM1zo` should do the Extensions.* projects for another year.

It's a bit of an annoying system to maintain; we're moving to another approach for work projects (use an agent to harvest GitHub releases and centrally update NuGet etc.), it might be the way to go for Serilog eventually, too.

**Pull request closed without merging**